### PR TITLE
Change Graylog version and add new parameters

### PR DESCRIPTION
**Pull request by oatenter**

### CODE REVIEW

**Review by 3d-pro — APPROVED** [review]

File: tasks/graylog_post.yml [hunk 1]
```diff
@@ -1,4 +1,15 @@
 ---
+- name: Set fact variable for list each items
+  set_fact:
+    graylog_elasticsearch_facts: "http://{{ item }}:{{ graylog_elasticsearch_port }}"
+  with_items: "{{ graylog_elasticsearch_hosts }}"
+  register: output_graylog_elasticsearch_facts
```
Comment (winggundamth) [inline tasks/graylog_post.yml]: Variable should prefix with graylog_

File: tasks/graylog_post.yml [hunk 2]
```diff
@@ -1,4 +1,15 @@
...
+
+- name: Set fact variable for all host Elasticsearch
+  set_fact:
+    graylog_elasticsearch_host: "{{ output_graylog_elasticsearch_facts.results | map(attribute='ansible_facts.graylog_elasticsearch_facts') | list }}"
+  when: output_graylog_elasticsearch_facts.results is defined
```
Comment (winggundamth) [inline tasks/graylog_post.yml]: Variable should prefix with graylog_

File: tasks/graylog_post.yml [hunk 3]
```diff
@@ -7,4 +18,15 @@
     group: root
     mode: 0644
   when: graylog_config_check.stat.exists
-  notify: Restart Graylog
\ No newline at end of file
+  changed_when: yes
+  notify: Restart Graylog
+
+- name: Copy Graylog configuration file in case of host doesn't have config file
+  template:
```
Comment (winggundamth) [inline tasks/graylog_post.yml:37]: What happen if templates/server.conf.j2 not found?

**Review by winggundamth — COMMENTED** [review]
Please check comment

**Pull request merged**